glide.lock

```diff
@@ -1,18 +1,19 @@
 hash: 620ed7b5a97e5e1c3f144f49863859862222390b5e00249714aab0b1d4c581c7
-updated: 2017-04-07T17:26:49.573117984-04:00
+updated: 2017-04-11T15:02:41.785025807-04:00
 imports:
 - name: cloud.google.com/go
-  version: f4dfefc998d92c53913c2ae5bc7cd3fba18e9e4b
+  version: 70fd9bf9d3537b799ae260d468f1ef7bf6afeaf1
   subpackages:
   - compute/metadata
+  - iam
   - internal
   - internal/optional
   - internal/version
   - storage
 - name: github.com/abourget/teamcity
   version: 6dde447fa54bc5b08b1a7bb1b85e39089cf27fb1
 - name: github.com/Azure/azure-sdk-for-go
-  version: 5b0aa1d05fbf518373abb366951e3587cc1b3154
+  version: 088007b3b08cc02b27f2eadfdcd870958460ce7e
```

Comment:
Sadly it's somewhat hard to avoid these bumps at the moment (though I could pin it, I guess, at the cost of another CI cycle).

The diff in this case seems uninteresting https://github.com/Azure/azure-sdk-for-go/compare/5b0aa1d05fbf518373abb366951e3587cc1b3154...088007b3b08cc02b27f2eadfdcd870958460ce7e